### PR TITLE
Feature/code clean

### PR DESCRIPTION
**Pull request by ghost**
- make more stuff static
- keep some defines local
- move some globals where they belong

### CODE REVIEW

File: src/confread.c [hunk 1]
```diff
@@ -51,40 +51,42 @@
...
-char    cToken[MAX_TOKEN_LENGTH];   // Token buffer...
-short   validToken;
+static struct {
+    FILE            *confFilePtr;       // File handle pointer
+    char            *iBuffer;           // Inputbuffer for reading...
+    unsigned int    bufPtr;             // Buffer position pointer.
+    unsigned int    readSize;           // Number of bytes in buffer after last read...
+    char    cToken[MAX_TOKEN_LENGTH];   // Token buffer...
+    short   validToken;
+} conf;
```
Comment (pali) [inline src/confread.c]: Is putting all variables into struct really needed? They are private for confread.c file, so `static` should be enough.

**Review by pali — COMMENTED** [review]
You can squash two commits which adds `static` keyword into one commit.

File: src/confread.c (hunk 1 above)
Comment (ghost) [reply src/confread.c]: ok

**Review by pali — CHANGES_REQUESTED** [review]
Seems that this pull request is broken as it contains broken merge. Please rebase and fix it.

**Comment by ghost** [comment]
this does not good in a direction i intended, i am not interested anymore to pull this changes

**Pull request closed without merging**